clients/cli/src/main.rs

```diff
@@ -112,6 +113,12 @@ enum Lockbook {
         path: String,
     },
 
+    /// Print the file tree with a given file as the head
+    PrintTree {
```

Comment:
I think we just want this to be called `Tree`. We may put this in a `debug` subcommand, not sure yet, but `Print` is redundant.